src/extension.ts

```diff
@@ -62,6 +68,53 @@ const provideInitialConfigurations = (): string => {
   ].join('\n');
 };
 
+const createTizenProject = async(): Promise<any> => {
+  fs.stat(lastModified.filePath, (err, stat) => {
+    if (lastModified.mtime < stat.mtime) {
+      getPath();
+      setup();
```

Comment:
Please remove this line since the `getPath` function executes the `setup` function.